cmd/reports-controller/main.go

```diff
@@ -245,6 +245,7 @@ func main() {
 	// ELSE KYAML IS NOT THREAD SAFE
 	kyamlopenapi.Schema()
 	// informer factories
+	kubeInformer := kubeinformers.NewSharedInformerFactory(kubeClient, resyncPeriod)
```

Comment:
We don't need this one right ?

Reply:
why?

Reply:
Are we using it ?

Reply:
I thought that would be needed if I call `StartInformersAndWaitForCacheSync`